src/iks001_cantileverBeamOneDGrid.cpp

```diff
@@ -35,57 +35,57 @@ Eigen::MatrixXd TimoshenkoBeamStiffness(auto localView, auto gridElement, auto q
   Ikarus::LocalBasis basisW(localView.tree().child(_0).finiteElement().localBasis());
   Ikarus::LocalBasis basisPhi(localView.tree().child(_1).finiteElement().localBasis());
 
-  // Determinant of Jacobian, obtained from gridElement
+  /// Determinant of Jacobian, obtained from gridElement
```

Comment:
Should we use here the variable name "gridElement" or write prosa "grid element" ?

Reply:
This applies also to similar question everywhere in ikarus

Reply:
We can write the prosa format while declaring the variable and later use the variable names in the comments and in the documentation for easier readability of the code.